background/function/main_test.py

```diff
@@ -11,3 +11,57 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import base64
+import json
+import os
+import pytest
+
+from google.cloud import firestore
+
+os.environ['GOOGLE_CLOUD_PROJECT'] = FIRESTORE_PROJECT_ID
```

Comment:
Where does this variable come from? Will the credentials path below work for anyone? Can it come from an environment variable (set in https://github.com/GoogleCloudPlatform/getting-started-python/blob/master/.kokoro/system_tests.sh)?

Reply:
No, it doesn't seem to pick up on variables set in system_tests.sh, they need to be set in Kokoro configs. It is now being set there.